### ISSUE TITLE
International numbers not supported

### ISSUE THREAD

**Issue by ghost** [twilio/OpenVBX-iPhone#3]
Numbers default to the US format. 
phoneNumber doesn't accept more than 10 digits.

**Comment by mikeseeh** [comment]
Correct. It would be great if this could be fixed, especially as Twilio does now its rollout in Europe.
